toolkit/Makefile

```diff
@@ -123,7 +123,8 @@ TLS_KEY     ?=
 
 # Build defines
 DIST_TAG           ?= .cm2
-BUILD_NUMBER       ?= $(shell git rev-parse --short HEAD)
+# Running 'git' as the owner of the repo, so it doesn't complain about the repo not belonging to root.
+BUILD_NUMBER       ?= $(shell runuser -u $$(stat -c "%U" $(PROJECT_ROOT)) -- git rev-parse --short HEAD)
```

Comment:
I think there was a workaround submitted by @anphel31 in the pipelines. We probably should address that as well :)

Reply:
Thanks for letting me know - I'll sync with @anphel31!